cli/config/configuration.toml.in

```diff
@@ -134,6 +134,13 @@ enable_iothreads = @DEFENABLEIOTHREADS@
 # used for 9p packet payload.
 #msize_9p = @DEFMSIZE9P@
 
+[factory]
+# VM templating support. Once enabled, new VMs are created from template
+# using vm cloning. They will share the same initial kernel memory by mapping
+# it readonly. When disabled, new VMs are created from scrash.
+#
+#enable_template = false
```

Comment:
As this is commented out, and I presume the default is therefore false/off, you may as well make the commented out line:
```
#enable_template = true
```
as that is the most likely line to be added or uncommented I think.

Reply:
Good point!